example/echo_c++/client.cpp

```diff
@@ -76,10 +76,10 @@ int main(int argc, char* argv[]) {
         stub.Echo(&cntl, &request, &response, NULL);
         if (!cntl.Failed()) {
             LOG(INFO) << "Received response from " << cntl.remote_side()
-                << " to " << cntl.local_side()
```

Comment:
Just keep the same indent style as that in other soruce code files？

Reply:
yep There is no other meaning.

Reply:
I mean it seems here the coding style was the same with other files. Why not just keeping what it was